crates/ide-completion/src/completions/postfix.rs

```diff
@@ -395,17 +395,21 @@ fn main() {
 }
 "#,
             expect![[r#"
-                sn box    Box::new(expr)
-                sn call   function(expr)
-                sn dbg    dbg!(expr)
-                sn dbgr   dbg!(&expr)
-                sn if     if expr {}
-                sn match  match expr {}
-                sn not    !expr
-                sn ref    &expr
-                sn refm   &mut expr
-                sn unsafe unsafe {}
-                sn while  while expr {}
+                sn box      Box::new(expr)
+                sn call     function(expr)
+                sn dbg      dbg!(expr)
+                sn dbgr     dbg!(&expr)
+                sn if       if expr {}
+                sn match    match expr {}
+                sn not      !expr
+                sn ref      &expr
+                sn refm     &mut expr
+                sn unsafe   unsafe {}
+                sn while    while expr {}
+                ex bar
+                ex false
+                ex foo(...)
+                ex true
```

Comment:
These are not field accesses / method calls so they shouldn't be shown here

Reply:
I'll remove completing dot access for now as getting it to consistently complete dot access needs some more work due to universal function call syntax it uses in some cases.